fluentgen/src/main/java/com/azure/autorest/fluent/model/clientmodel/FluentExampleLiveTestStep.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * // Copyright (c) Microsoft Corporation. All rights reserved.
+ * // Licensed under the MIT License.
+ */
+
+package com.azure.autorest.fluent.model.clientmodel;
+
+import com.azure.autorest.fluent.template.FluentExampleTemplate;
+
+public class FluentExampleLiveTestStep extends FluentLiveTestStep {
+
+    private FluentExampleTemplate.ExampleMethod exampleMethod;
+
+    public FluentExampleTemplate.ExampleMethod getExampleMethod() {
+        return exampleMethod;
+    }
+
+    public void setExampleMethod(FluentExampleTemplate.ExampleMethod exampleMethod) {
+        this.exampleMethod = exampleMethod;
+    }
```

Comment:
Personally prefer ctor with required parameters. I.e. prefer immutable class, if not, as many field as `final`. Do so if convenient.

Reply:
Agreed, will change it.

Reply:
Changed.